client/internal/url/url.go

```diff
@@ -76,3 +77,15 @@ func ReadSmartLicense(baseUrl string) string {
 func ReadAccessPolicies(baseUrl string, domainUid string, limit int) string {
 	return fmt.Sprintf("%s/fmc/api/fmc_config/v1/domain/%s/policy/accesspolicies?limit=%d", baseUrl, domainUid, limit)
 }
+
+func CreateUser(baseUrl string, username string) string {
+	return fmt.Sprintf("%s/anubis/rest/v1/users/%s", baseUrl, username)
+}
+
+func ReadUserByUsername(baseUrl string, username string) string {
+	return fmt.Sprintf("%s/anubis/rest/v1/users?q=name:%s", baseUrl, username)
+}
+
+func UserByUid(baseUrl string, uid string) string {
```

Comment:
maybe `ReadUserByUid`?

Reply:
It's used for both read and delete that's why